src/resources/classes/assetList.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Cognite AS
+
+import { chunk } from 'lodash';
+import { CogniteClient } from '../..';
+import {
+  CogniteEvent,
+  CogniteInternalId,
+  FilesMetadata,
+  GetTimeSeriesMetadataDTO,
+} from '../../types/types';
+import { EventsAPI } from '../events/eventsApi';
+import { FilesAPI } from '../files/filesApi';
+import { TimeSeriesAPI } from '../timeSeries/timeSeriesApi';
+import { Asset } from './asset';
+
+export class AssetList extends Array<Asset> {
+  private client: CogniteClient;
+  constructor(client: CogniteClient, items: Asset[]) {
+    super(...items);
+    this.client = client;
+  }
+
+  public delete = () => {
+    return this.client.assets.delete(this.map(asset => ({ id: asset.id })));
+  };
+
+  public timeSeries = async (): Promise<GetTimeSeriesMetadataDTO[]> => {
```

Comment:
I think you don't need to specify type here, as you do this `as GetTimeSeriesMetadataDTO[]` already